historical/common/sns.py

```diff
@@ -0,0 +1,98 @@
+"""
+.. module: historical.common.sns
+    :platform: Unix
+    :copyright: (c) 2018 by Netflix Inc., see AUTHORS for more
+    :license: Apache, see LICENSE for more details.
+.. author:: Mike Grima <mgrima@netflix.com>
+"""
+import logging
+import json
+import math
+import os
+import sys
+
+import boto3
+from retrying import retry
+
+from raven_python_lambda import RavenLambdaWrapper
+from historical.constants import CURRENT_REGION, REGION_ATTR, SNSZR_REGIONS
+
+log = logging.getLogger('historical')
+
+
+@retry(stop_max_attempt_number=4, wait_exponential_multiplier=1000, wait_exponential_max=1000)
+def _publish_message(client, blob, topic_arn):
+    client.publish(TopicArn=topic_arn, Message=blob)
+
+
+def shrink_sns_blob(record):
+    """
+    Makes a shrunken blob to be sent to SNS (due to size limitations of SNS messages).
+    This will essentially remove the "configuration" field such that
```

Comment:
Such that what?

Reply:
Fixing.  It's such that the SNS message size is under 256 KB.